packages/Graph/MSGraphClient.cs

```diff
@@ -14,14 +14,33 @@ namespace Microsoft.Bot.Components.Graph
     /// </summary>
     public class MSGraphClient
     {
+#if DEBUG
+        private static IGraphServiceClient clientOverride;
+
+        /// <summary>
+        /// Allows setting test instance of the graph service client
+        /// </summary>
+        /// <param name="client">The client that should be used</param>
+        public static void RegisterTestInstance(IGraphServiceClient client)
```

Comment:
Why did you have to add this? Were the tests not able to auto resolve the httpclient? I was under the impression that was the case

Reply:
They can. However, I wanted to actually isolate the code at the IGraphServiceClient layer rather than the lower HttpClient layer which depends on hand-crafting and updating the json, which can be more error prone.